fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaGenerator.java

```diff
@@ -628,6 +738,67 @@ public void addLogicalResources(PhysicalDataModel pdm) {
         pdm.addObject(tbl);
     }
 
+    /**
+     * Adds a table to support logical identity management of resources
+     * when using a distributed RDBMS such as Citus. It represents the
+     * mapping:
+     * 
+     * (RESOURCE_TYPE_ID, LOGICAL_ID) -> (LOGICAL_RESOURCE_ID)
+     * 
+     * LOGICAL_RESOURCE_ID values are assigned from the sequence FHIR_SEQUENCE.
+     * 
+     * When using Citus (or similar), this table is distributed by LOGICAL_ID,
+     * which means we can use a primary key of {RESOURCE_TYPE_ID, LOGICAL_ID}. 
+     * This is required to ensure that we can lock the logical resource to 
+     * avoid any concurrency issues.
+     * 
+     * LOGICAL_RESOURCE_IDENT records are also generated when the tuple 
+     * (RESOURCE_TYPE_ID. LOGICAL_ID) is used as a local resource reference 
+     * value. For example:
+     *   "reference": "Patient/aPatientId"
+     * will create a new LOGICAL_RESOURCE_IDENT record if the Patient resource
+     * "aPatientId" has not yet been created. The LOGICAL_RESOURCES record is 
+     * not created until the actual resource is created.
```

Comment:
This section is key.  Before reading this, I was wondering why we even needed this table when the same info is made available from LOGICAL_RESOURCES.

Reply:
but maybe update the `.` to a `,` in `(RESOURCE_TYPE_ID, LOGICAL_ID)` if you're in here

Reply:
Yep, and added a little more context regarding the Citus bit.